docs/sources/mimir/configure/configure-otel-collector.md

```diff
@@ -15,9 +15,9 @@ To send OTLP data to Grafana Cloud, refer to [Send data using OpenTelemetry Prot
 
 When using the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/), you can write metrics into Mimir via two options: `prometheusremotewrite` and `otlphttp`.
 
-We recommend using the `prometheusremotewrite` exporter when possible because the remote write ingest path is tested and proven at scale.
+We recommend using each protocol's respective exporter and native Mimir endpoint.
```

Comment:
Not sure exactly how to interpret this. Why not just recommend to use the `otlphttp` exporter, as it's now our recommendation?

```suggestion
We recommend using the `otlphttp` exporter when possible.
```

Reply:
The OTEL Collector and distributions like Alloy can also receive Prometheus metrics, and export Prometheus metrics. The article is titled "Configure the OTEL Collector", it does not appear to be exclusive to sending OTEL data. So Prometheus should go through Prometheus, and OTEL through OTEL.

We can get more specific, I left it off at first because I thought it sounded repetitive:

```markdown
We recommend using each protocol's respective exporter and native Mimir endpoint:
* Promtheus data through `prometheusremotewrite`
* `OTLP` data through `otlphttp`

Reply:
Thanks for the explanation, I don't think I was aware of OTel Collector being able to receive Prometheus metrics. I think I prefer your more explicit version.

> `OTLP` data through `otlphttp`

I think "OTel data through `otlphttp`" makes more sense, since the data itself is OTel format and OTLP is just the wire protocol.

Reply:
updated